english/explore-outside/investigate-marshmallow-structure/greet-green/examine_wall/contemplate/contemplate.md

```diff
@@ -0,0 +1 @@
+You sit cross-legged in front of the wall and ask yourself a variety of questions, such as "Why am I here", and "How come all the good bands were before my time".  After an indeterminable time, Morgan Freeman rests his hand on your shoulder and asks you, "Are you ready to get busy living?"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.